UI/MainForm.cs

```diff
@@ -94,6 +94,9 @@ public MainForm()
             // finally set up logging (based on settings)
             InitializeLogging();
 
+            // Initialize the board configurations
+            BoardDefinitions.Load();
```

Comment:
Board definitions get loaded immediately after logging is set up and before any other work is done.